src/doc/src/reference/registries.md

```diff
@@ -45,6 +47,18 @@ CARGO_REGISTRIES_MY_REGISTRY_INDEX=https://my-intranet:8080/git/index
 > Note: [crates.io] does not accept packages that depend on crates from other
 > registries.
 
+### Sparse registries
+Cargo supports remote registries served over plain HTTP rather than git. These
+registries can be identified by urls starting with `sparse+http://` or `sparse+https://`.
+
+When fetching index metadata over the sparse HTTP protocl, Cargo only downloads
+the metadata for relevant crates, which can save significant time and bandwidth.
+
+The format of the sparse index is identical to a checkout of a git-based index.
```

Comment:
I think this may need a few more sentences for people to get the idea.

Reply:
Does this make more sense?

Reply:
That is a big step forward, thank you.